api-reference/media-api/delete-file-version.md

```diff
@@ -0,0 +1,62 @@
+# Delete file version
+
+{% swagger baseUrl="https://api.imagekit.io" path="/v1/files/:fileId/versions/:versionId" method="delete" summary="Delete file version API" %}
+{% swagger-description %}
+Delete any non-current version of a file.
+{% endswagger-description %}
+
+{% swagger-parameter in="path" name="fileId" type="string" required="true" %}
+The unique fileId of the uploaded file. `fileId` is returned in list files API and upload API.
+{% endswagger-parameter %}
+
+{% swagger-parameter in="path" name="versionId" type="string" required="true" %}
+The unique versionId of the uploaded file's version. This is returned in list files API and upload API as `id` within the `versionInfo` parameter.
+{% endswagger-parameter %}
+
+{% swagger-parameter in="header" name="Authorization" type="string" required="true" %}
+base64 encoding of `your_private_api_key:`
+
+**Note the colon in the end.**
+{% endswagger-parameter %}
+
+{% swagger-response status="204" description="" %}
+```
+```
+{% endswagger-response %}
+
+{% swagger-response status="400" description="If the requested file version to delete is the current version of file" %}
+```javascript
+{
+     "message" : "You cannot delete current version of a file. Please restore file to a different version to delete this version.",
```

Comment:
`Please restore the file to a different version to delete this version` is confusing; we can remove this because `You cannot delete the current version of a file` alone is sufficient.